corehq/apps/app_manager/util.py

```diff
@@ -153,18 +153,17 @@ def change_xmlns(xform, replacing):
 
     form.source = xml
 
-    # For registration forms, assume that the first question is the case name
+    # Assume that the first question is the case name
     # unless something else has been specified
-    if toggles.APP_MANAGER_V2.enabled(app.domain):
```

Comment:
Was removing this check intentional?  The rest of the changes switch domain to user, but this one drops a toggle check.